<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #1E1E1E;
}

QWidget {
    background-color: #252525;
    color: #E0E0E0;
}

QLabel {
    color: #E0E0E0;
}

QLineEdit {
    background-color: #2D2D2D;
    color: #E0E0E0;
    border: 1px solid #333;
    border-radius: 6px;
    padding: 8px;
}

QPushButton {
    background-color: #34495E;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 8px 24px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #2C3E50;
}

QPushButton:pressed {
    background-color: #1F2B38;
}</string>
  </property>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Придумайте надёжный (!) пароль: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Для регистрации ввведите данные:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Придумайте логин:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>221</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Регистрация, вперёд!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Подтвердите ваш пароль:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nicknameEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Придумайте отоброжаемое имя:</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
